app/models/enum/queries.py

```diff
@@ -4,3 +4,7 @@
 class QueryFormat(str, Enum):
     json = "json"
     csv = "csv"
+
+class CsvDelimiter(str, Enum):
+    comma = ","
+    tab = "\t"
```

Comment:
Just realizing that there is already such an Enum

https://github.com/wri/gfw-data-api/blob/master/app/models/enum/creation_options.py#L26-L31